src/__tests__/renderer.test.tsx

```diff
@@ -369,16 +369,14 @@ describe('slack jsx', () => {
   });
 
   it('renders inline text elements with promises returning text elements', async () => {
-    const renderText = async () => {
-      const strings = await Promise.all(
-        ['foo', 'bar'].map(s => render(<Mention userId={s} />))
-      );
-      return `world! ${strings}`;
+    const renderText = () => {
+      const strings = ['foo', 'bar'].map(s => <Mention userId={s} />);
+      return <>world! {strings}</>;
     };
 
     const message = (
       <ContextBlock>
-        <MarkdownText>Hello, {await renderText()}</MarkdownText>
+        <MarkdownText>Hello, {renderText()}</MarkdownText>
```

Comment:
Just curious - If we are not doing async `renderText` what would be the purpose of this test be?

Reply:
down below he does `await render(message)` for the entire message

Reply:
just to demonstrate how to define components that return text nodes